src/pages/utilities/gestures.md

```diff
@@ -0,0 +1,208 @@
+---
+previousText: ''
+previousUrl: ''
+nextText: 'Platform'
+nextUrl: '/docs/utilities/platform'
+---
+
+# Gestures
+
+## Overview
+
+Ionic Gestures is a utility that allows developers to build custom gestures and interactions for their application in a platform agnostic manner. Developers do not need to be using a particular framework such as React or Angular, nor do they even need to be building an Ionic app! As long as developers have access to `@ionic/core`, they will have access to all of Ionic Gestures.
```

Comment:
Should this be `Ionic Gestures are`? It just reads strangely to me. Same with the Animations.

Reply:
Ionic Gestures is the name of the package so I think it's supposed to be "is" since it's a singular utility